### PR TITLE
Deposit insurance early withdrawal

### PR DESCRIPTION
**Pull request by aaalata**
Hi!
Please do the code review.
Thanks!

### CODE REVIEW

File: src/test/java/pl/edu/agh/iet/katabank/steps/DepositManagementSteps.java [hunk 1]
```diff
@@ -35,8 +35,10 @@
     private Set<Deposit> customerDeposits;
     private LocalDate date;
     private BigDecimal amount;
+    private BigDecimal depositedAmount;
     private InterestPolicy interestPolicy;
     private DepositDurationDetails durationDetails;
+    LocalDate withdrawalDate;
```
Comment (anthonyflea) [inline src/test/java/pl/edu/agh/iet/katabank/steps/DepositManagementSteps.java]: Since it's used only in one method, use local variable instead

**Review by anthonyflea — APPROVED** [review]
Code review done.
Please merge to master.

File: src/test/java/pl/edu/agh/iet/katabank/steps/DepositManagementSteps.java (hunk 1 above)
Comment (aaalata) [reply src/test/java/pl/edu/agh/iet/katabank/steps/DepositManagementSteps.java]: Right. I used in another test previously. Now it should be local.
Corrected.

**Comment by aaalata** [comment]
done

**Pull request merged**